### PR TITLE
do not docker-ignore package-lock

### PR DESCRIPTION
**Pull request by jellegerbrandy**

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -1,5 +1,4 @@
 node_modules
-package-lock.json
 *.log
```
Comment (jellegerbrandy) [inline .dockerignore]: including package-lock in the docker file will make our builds slightly more predicable

File: Dockerfile [hunk 2]
```diff
@@ -12,7 +11,7 @@ RUN chmod +x /wait /entry
 
 WORKDIR /usr/app
 COPY . .
-RUN npm install
+RUN npm ci
```
Comment (orenyodfat) [inline Dockerfile]: what ci stand for ?

Comment (jellegerbrandy) [reply Dockerfile]: "continuous integration"

Comment (jellegerbrandy) [reply Dockerfile]: https://docs.npmjs.com/cli/ci

**Pull request merged**